uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences.js

```diff
@@ -0,0 +1,64 @@
+var Promise = require('es6-promise').Promise;
```

Comment:
there is generic modals folder for all modals so i think it will be better to keep it there only